src/main/content/_assets/css/header.css

```diff
@@ -1,13 +1,27 @@
 /* NAVIGATION BAR */
 
+main {
+    margin-top: -60px;
+}
+
 .navbar {
     padding: 10px 0px 10px 0px;
     width: 80%;
     margin: 0 auto;
+    z-index: 9999;
+}
+
+.bg-custom {
+    background-color: transparent !important;
```

Comment:
Is the bg-custom with !important needed with the negative margin on the <main> to make the navbar transparent?

Reply:
Hmm I don't think that's needed so I can remove that.